scripts/bert/index.rst

```diff
@@ -52,5 +52,25 @@ It gets RTE validation accuracy of `70.8% <https://raw.githubusercontent.com/dml
 It gets MNLI validation accuracy ,On dev_matched.tsv: 84.6%
 On dev_mismatched.tsv: 84.7%. `log <https://github.com/dmlc/web-data/blob/master/gluonnlp/logs/bert/finetuned_mnli.log>`_
 
-
 Some other tasks can be modeled with `--task_name` parameter.
+
+Pre-training with BERT
+~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
remember to report the downstream task results from finetuning the pre-trained model from this.